CONTRIBUTING.rst

```diff
@@ -154,7 +219,7 @@ You can use the default Breeze configuration as follows:
 
 .. code-block:: bash
 
-   ./breeze --initialize-local-virtualenv
+   ./breeze initialize-local-virtualenv
```

Comment:
Is this a related change?

Reply:
I think not related but I would not obiect fixing it.

Reply:
Not related, just a typo I noticed when trying to use Breeze for the first time. If no one opposes, I'll keep it here.